.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,20 @@
-
 # Before you proceed
 
 - Read the [wiki](https://github.com/ckb-next/ckb-next/wiki).
-- Try the latest code by building the software manually.
 - Search in open and closed [Issues](https://github.com/ckb-next/ckb-next/issues).
+- If possible, try the latest code by building the software manually.
 - File unrelated issues separately.
 - Use [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to format the message.
-- There are 3 types of issues: new device, feature, bug. Use only one. Delete redundant sections before sending.
-
+- There are 3 types of issues: new device, feature, bug.
+Use only one. Delete redundant sections before sending.
+If your issue doesn't fit in any of those categories, please use the *Something else* field at the bottom.
+- **Click the `Generate Report` button in the `Settings` tab and attach the generated report to this issue.**
 ---
 
-# Devices information
-
-Open a terminal and execute:
-```
-curl https://raw.githubusercontent.com/ckb-next/ckb-next/master/ckb-next-dev-detect 2>/dev/null | bash
-```
-
-Upload `ckb-next-dev-detect-report.gz` with this issue.
-
----
 
 # New device support request
 
-Upload:
-  * Linux:
-    * `ckb-next-dev-detect-report.gz` from the section above
-  * macOS:
-    * `ckb-next-dev-detect-report.gz` from the section above
-    * full USB bus dump by following [the instructions](https://github.com/mattanger/ckb-next/issues/31#issuecomment-285380447)
+Write the name of the device and attach the report generated above.
```

Comment:
Probably better to explicitly say: "Enter the name of your device here and attach the report generated above."

Reply:
Sounds good